placement/application.go

```diff
@@ -0,0 +1,58 @@
+package placement
```

Comment:
I didn't name it to `opportunities` because we also have `preference` structs here ... so Placement seem like an umbrella term for `opportunities` `applications` `preference`

Reply:
The name of the service is `opportunities` microservice and if thats the case divide the code in multiple packages